java/src/main/java/com/amido/stacks/workloads/menu/handlers/CreateItemHandler.java

```diff
@@ -1,61 +1,29 @@
 package com.amido.stacks.workloads.menu.handlers;
 
 import com.amido.stacks.workloads.menu.commands.CreateItemCommand;
-import com.amido.stacks.workloads.menu.domain.Category;
-import com.amido.stacks.workloads.menu.domain.Item;
 import com.amido.stacks.workloads.menu.domain.Menu;
-import com.amido.stacks.workloads.menu.exception.CategoryDoesNotExistException;
-import com.amido.stacks.workloads.menu.exception.ItemAlreadyExistsException;
-import com.amido.stacks.workloads.menu.repository.MenuRepository;
-import java.util.ArrayList;
-import java.util.List;
+import com.amido.stacks.workloads.menu.service.v1.ItemService;
 import java.util.Optional;
 import java.util.UUID;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
 
+@RequiredArgsConstructor
 @Component
 public class CreateItemHandler extends MenuBaseCommandHandler<CreateItemCommand> {
 
-  public CreateItemHandler(MenuRepository menuRepository) {
-    super(menuRepository);
-  }
+  ItemService itemService;
```

Comment:
Possibly want a private or protected on this?

Reply:
fixed